smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/SmithyCoreSubmodules.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package software.amazon.smithy.typescript.codegen;
+
+public final class SmithyCoreSubmodules {
```

Comment:
Suggestion: doc comment on what this class contains, the '/' was slightly confusing to me at first, until I saw the usage in `addImportSubmodule`

Reply:
added docblock